### PR TITLE
Add bun package manager

### PR DESCRIPTION
**Pull request by ludicast**
[Bun](https://bun.sh) is definitely growing and dependency-cruiser works fine with it locally.  This adds it to your list of acceptable package managers - please let me know if I'm forgetting anything.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,9 +72,9 @@ with:
   cruise-script: yarn run -s depcruise --exclude "^lib"
 ```
 
-### Usage with `npm` or `pnpm`
+### Usage with `bun`, `npm` or `pnpm`
```
Comment (ludicast) [inline README.md:75]: was adding alphabetically, `bun` can be wherever you want obviously.

File: README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ jobs:
       - uses: MH4GF/dependency-cruiser-report-action@v2
 ```
 
-## Adbanced Usage
+## Advanced Usage
```
Comment (MH4GF) [inline README.md:41]: wow, thanks!

**Comment by MH4GF** [comment]
@ludicast 
CI is down, but it seems to be a problem that also occurs in the main branch. Please take this one in.

- https://github.com/MH4GF/dependency-cruiser-report-action/pull/275

**Comment by ludicast** [comment]
> @ludicast CI is down, but it seems to be a problem that also occurs in the main branch. Please take this one in.
> 
> * [fix: Remove packageManager field from package.json #275](https://github.com/MH4GF/dependency-cruiser-report-action/pull/275)

cool - caught up to main :crossed_fingers:

**Comment by MH4GF** [comment]
https://github.com/MH4GF/dependency-cruiser-report-action/actions/runs/6989311066/job/19018112442

CI continues to fail... but this is a problem that pull requests from forked repositories must be `on: pull_request_target` to work.
There is a security concern, but this repository is not frequently maintained, so it will not be changed this time.

**Review by MH4GF — APPROVED** [review]
LGTM! Thanks! 😄

**Comment by MH4GF** [comment]
It works 👍🏻 

- https://github.com/MH4GF/dependency-cruiser-report-action/pull/280#issuecomment-1826338862

**Comment by MH4GF** [comment]
[v2.3.0](https://github.com/MH4GF/dependency-cruiser-report-action/releases/tag/v2.3.0) 🚀

**Pull request merged**